.env.test

```diff
@@ -14,6 +14,7 @@ METAPHYSICS_PRODUCTION_ENDPOINT=https://metaphysics-production.artsy.net
 METAPHYSICS_STAGING_ENDPOINT=https://metaphysics-staging.artsy.net
 POSITRON_API_BASE=https://writer.artsy.test/api
 PREDICTION_ENDPOINT=https://live.artsy.net
+OPENREDIS_URL=redis://localhost:6379/1
```

Comment:
Seems like we need to provide this now when running locally.

Reply:
It’s already in there? (Except for that `/1` path)  https://github.com/artsy/metaphysics/blob/fe0e40b131d8cab325268dbcb358abf3846cf6d0/.env.test#L12

Reply:
Hmm, somehow that didn't work for me...I'll retry.